applications/views.py

```diff
@@ -106,12 +106,8 @@ def applications_csv(request, city):
     rsvp_status = request.GET.getlist('rsvp_status', None)
     page = get_event_page(city, request.user.is_authenticated(), False)
     order = request.GET.get('order', None)
-    try:
-        applications = get_applications_for_page(
-            page, state, rsvp_status, order).prefetch_related('answer_set')
-    except:
-        return redirect('core:event', city=city)
-
+    applications = get_applications_for_page(
```

Comment:
We should still have exception handling here, - `Form.DoesNotExist` is raised by the utility function so that should still be handled here.